core/lib/l1_contract_interface/src/i_executor/commit/kzg.rs

```diff
@@ -102,6 +102,20 @@ impl KzgInfo {
         res
     }
 
+    pub fn to_blob_commitment(&self) -> [u8; 32] {
+        let mut commitment = [0u8; 32];
+        let hash = &Keccak256::digest(
+            [
+                self.versioned_hash.to_vec(),
+                self.opening_point[16..].to_vec(),
+                self.opening_value.to_vec(),
+            ]
```

Comment:
```suggestion
            [
                self.versioned_hash.as_slice(),
                &self.opening_point[16..],
                self.opening_value.as_slice(),
            ]
```

additional clones are likely unnecessary here.